packages/www/components/Dashboard/Chart/index.tsx

```diff
@@ -65,26 +93,45 @@ const Chart = ({ data }) => {
         Seconds since stream loaded
       </Text>
       <ResponsiveContainer width="99%" height={300}>
-        <AreaChart data={data}>
-          <defs>
-            <linearGradient id="colorKbps" x1="0" y1="0" x2="0" y2="1">
-              <stop offset="5%" stopColor="#6E56CF" stopOpacity={0.8} />
-              <stop offset="95%" stopColor="#6E56CF" stopOpacity={0} />
-            </linearGradient>
-          </defs>
-          <XAxis dataKey="name" />
+        <LineChart>
+          <XAxis
+            type="number"
+            dataKey="name"
+            domain={[
+              data[0]?.name,
+              data[data.length - 1].name === 0
+                ? 10
+                : data[data.length - 1].name,
```

Comment:
Is this the same as checking if there is more than 1 element in the `data` array? If so, maybe this is clearer:
```suggestion
              data.length > 1
                ? 10
                : data[data.length - 1].name,
```
Additionally, what does this `10` mean?

Reply:
This is the end value of X-Axis when loads init chart data
![Screenshot](https://user-images.githubusercontent.com/35409129/159534218-6d9071c8-92cb-4cd2-a27e-b687ce0f225a.png)